indicators/microsoft-fyfcvk8e.yaml

```diff
@@ -0,0 +1,23 @@
+title: Microsoft Phishing Kit fyfcvk8e
+description: |
+    Detects a Microsoft phishing kit with a hardcoded MFA phone number and misspelled words.
+
+references:
+    - https://urlscan.io/result/0f35c05b-73e0-4397-9e7e-9e3edb508d16
+    - https://urlscan.io/result/e73ca666-5a09-4c0e-949b-33a8f6ee7564
```

Comment:
Interestingly, after investigating this kit, it seems like there is some PhaaS platform being used to deploy this. Because in the code that is returned by the `sc.php` page, it seems to be doing some sort of license checking to see if the deployer actually has a valid API key for the PhaaS platform.

Looking further into the code returned by `sc.php` it appears to do some cryptography to decrypt and then load the phishing page content. Prior to this however it requires a valid response from the license server.

A very convoluted yet unique way to, I suppose, protect the business of the PhaaS platform from people trying to *take* their *work*

[Here](https://urlscan.io/search/#filename%3A%22sc.php%22%20AND%20filename%3A%22jquery-3.1.1.min.js%22%20AND%20filename%3A%22crypto-js.min.js%22) is a URLScan query to potentially identify other occurences (nearly 1,700) of this specific kit.

Reply:
@IlluminatiFish see the latest commit [9f11a37](https://github.com/phish-report/IOK/pull/228/commits/9f11a37f3c32f8589ecf03ddc4d73e302c32fa80) including the network call for sc.php. I included the cryptojs and jquery calls as well to better match the query for this specific kit.

Reply:
LGTM!